app/assets/stylesheets/course/announcements.css.scss

```diff
@@ -0,0 +1,25 @@
+// Place all the styles related to the course/announcements controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+@import "variables";
+@import "mixins/future";
+
+.course-announcements {
+  &.index {
+    .announcement {
+      padding-left: 8px;
```

Comment:
I think using `px` is ok here, as long as the layout looks fine on mobile, tablet and desktop. I'm not extremely clear about when to use relative vs absolute widths for layout sizes. But to my knowledge, Bootstrap only uses `em` in their font-related styles properties. They use `px` for padding and margins of the grids and other UI components.